### PR TITLE
[PR] Layered: Fix too little padding reserved for compound node labels

### PR DESCRIPTION
**Pull request by le-cds**

### CODE REVIEW

File: plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java [hunk 1]
```diff
@@ -479,7 +479,8 @@ private LGraph createLGraph(final ElkNode elkgraph) {
         // since size information stored there may apply to the current graph node)
         ElkPadding nodeLabelpadding = NodeLabelAndSizeCalculator.computeInsideNodeLabelPadding(
                 elkgraph.getParent() == null ? null : ElkGraphAdapters.adapt(elkgraph.getParent()),
-                ElkGraphAdapters.adaptSingleNode(elkgraph));
+                ElkGraphAdapters.adaptSingleNode(elkgraph),
```
Comment (uruuru) [inline plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java:482]: Formatting looks off :).

Comment (le-cds) [reply plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java:482]: Oh, how so? I tried to put each argument on a separate line.

Comment (uruuru) [reply plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java:482]: Usually you would still increase the indentation after the first line break. At least that's what feels natural to me.

Comment (le-cds) [reply plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java:482]: Oh, interesting, I've never done it that way. 😀

Comment (uruuru) [reply plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java:482]: Actually, I misread the diff. You added a parameter to the `computeInsideNodeLabelPadding` method, not the `adaptSingleNode`. Hence the indentation I was referring to happens in line 481.

**Pull request merged**